docs/examples/frameworks/pytorch_setup/README.rst

```diff
@@ -1,77 +1,88 @@
+.. NOTE: This file is auto-generated from examples/frameworks/pytorch_setup/index.rst
+.. This is done so this file can be easily viewed from the GitHub UI.
+.. **DO NOT EDIT**
+
 .. _pytorch_setup:
 
 PyTorch Setup
 =============
 
 **Prerequisites**: (Make sure to read the following before using this example!)
 
+
+The full source code for this example is available on `the mila-docs GitHub
+repository.
+<https://github.com/mila-iqia/mila-docs/tree/master/docs/examples/frameworks/pytorch_setup>`_
+
+
 * :ref:`Quick Start`
 * :ref:`Running your code`
 * :ref:`Conda`
 
 
+
 **job.sh**
 
 
 .. code:: bash
 
-   #!/bin/bash
-   #SBATCH --gres=gpu:1
-   #SBATCH --cpus-per-task=1
-   #SBATCH --mem=16G
-   #SBATCH --time=00:15:00
-   #SBATCH --partition=unkillable
+    #!/bin/bash
+    #SBATCH --gres=gpu:1
+    #SBATCH --cpus-per-task=1
+    #SBATCH --mem=16G
+    #SBATCH --time=00:15:00
+    #SBATCH --partition=unkillable
```

Comment:
Are we changing to 4 spaces? We should probably have single huge commit changing all 3 spaces to 4 spaces. Else, we should stick to 3 spaces

Reply:
4 spaces. And I'd be in favour of merging the PR as-is, if you don't mind.

Reply:
Sounds like a one man decision ;). I would argue that style changes should be done project wide but if we're eager to merge this one I suppose we can take a shortcut here. I would still prefer for this kind of style modification, which is very subjective, to be done in a separate PR in the future. (also, not talking about python code indentation, for which 4 is the standard, only for rst, for which 3 spaces is used pretty much everywhere)

Reply:
Added an issue to resolve this question properly : https://github.com/mila-iqia/mila-docs/issues/208

Reply:
The fixed requirement here is that all code blocks in the examples (.diffs or not) need to be directly pasteable into an editor and land correctly indented (4 spaces).
I'm fine with whatever solution that addresses this requirement.